gateway/api/services/file_storage.py

```diff
@@ -171,3 +171,33 @@ def get_file(self, file_name: str) -> Optional[Tuple[FileWrapper, str, int]]:
             file_size = os.path.getsize(path_to_file)
 
             return file_wrapper, file_type, file_size
+
+    def remove_file(self, file_name: str) -> bool:
+        """
+        This method returns a file from file_name:
+            - Only files with supported extensions are available to download
+            - It returns only a file from a user or a provider file storage
+
+        Returns:
+            FileWrapper: the file itself
+            str: with the type of the file
+            int: with the size of the file
+        """
+
+        file_name_path = os.path.basename(file_name)
+        path_to_file = sanitize_file_path(os.path.join(self.file_path, file_name_path))
+
+        try:
```

Comment:
As we talked, I'm going to open a PR doing a refactor in the rest of the methods to start using try/except approaches.